src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -267,74 +321,67 @@ export const WorkspaceForm = ({
-              </EuiTitle>
-            </>
-          }
-        >
-          <EuiFlexGrid style={{ paddingLeft: 20, paddingTop: 20 }} columns={2}>
-            {featureOrGroups.map((featureOrGroup) => {
-              const features = isWorkspaceFeatureGroup(featureOrGroup)
+      </EuiPanel>
+      <EuiSpacer />
+      <EuiPanel>
+        <EuiTitle size="s">
+          <h2>Workspace features</h2>
+        </EuiTitle>
+        <EuiFlexGrid style={{ paddingLeft: 20, paddingTop: 20 }} columns={2}>
+          {featureOrGroups.map((featureOrGroup) => {
+            const features = isWorkspaceFeatureGroup(featureOrGroup) ? featureOrGroup.features : [];
+            const selectedIds = selectedFeatureIds.filter((id) =>
+              (isWorkspaceFeatureGroup(featureOrGroup)
                 ? featureOrGroup.features
-                : [];
-              const selectedIds = selectedFeatureIds.filter((id) =>
-                (isWorkspaceFeatureGroup(featureOrGroup)
-                  ? featureOrGroup.features
-                  : [featureOrGroup]
-                ).find((item) => item.id === id)
-              );
-              return (
-                <EuiFlexItem key={featureOrGroup.name}>
-                  <EuiCheckbox
-                    id={
-                      isWorkspaceFeatureGroup(featureOrGroup)
-                        ? featureOrGroup.name
-                        : featureOrGroup.id
-                    }
-                    onChange={
-                      isWorkspaceFeatureGroup(featureOrGroup)
-                        ? handleFeatureGroupChange
-                        : handleFeatureCheckboxChange
-                    }
-                    label={`${
-                      featureOrGroup.name === 'OpenSearch Plugins'
-                        ? 'OpenSearch Features'
-                        : featureOrGroup.name
-                    }${features.length > 0 ? `(${selectedIds.length}/${features.length})` : ''}`}
-                    checked={selectedIds.length > 0}
-                    indeterminate={
-                      isWorkspaceFeatureGroup(featureOrGroup) &&
-                      selectedIds.length > 0 &&
-                      selectedIds.length < features.length
-                    }
+                : [featureOrGroup]
+              ).find((item) => item.id === id)
+            );
+            return (
+              <EuiFlexItem key={featureOrGroup.name}>
+                <EuiCheckbox
+                  id={
+                    isWorkspaceFeatureGroup(featureOrGroup)
+                      ? featureOrGroup.name
+                      : featureOrGroup.id
+                  }
+                  onChange={
+                    isWorkspaceFeatureGroup(featureOrGroup)
+                      ? handleFeatureGroupChange
+                      : handleFeatureCheckboxChange
+                  }
+                  label={`${
+                    featureOrGroup.name === 'OpenSearch Plugins'
```

Comment:
Could you help to revert this change? It's just a hack for demo, we don't need it now.

Reply:
Removed